### DISCUSSION TITLE
State for matrix As

### DISCUSSION THREAD

**Discussion by Leonawyciii** [CURENT/andes#450] [Q&A] [answered]
Hi Dr. Cui and other developers,

when I get the matrix with ss.EIG.As, how do I get each state of the A matrix?  Does ss.EIG.x_name show each state corresponding to the A matrix?  or I can find the state in the section EIGENVALUE DATA in the file of SMIB_eig.txt.
Also, when I converge the As from andes to Matlab, how to know the state corresponding to each row of the A matrix?

**Comment by cuihantao** [comment, marked as answer]
Hello, the states corresponding to each row is in `ss.EIG.x_name`. I'm not sure about the differences in your questions.

**Reply by Leonawyciii** [reply]
Thank you for your answer. It is helpful. In the _eig.txt file, the EIGENVALUE DATA also shows some values with status. What is the meaning of these data?

<img width="880" alt="Screen Shot 2023-05-05 at 10 56 24 am" src="https://user-images.githubusercontent.com/116365865/236494289-fd52542d-76f4-40a1-8315-ce7ed8756fa0.png">

**Reply by cuihantao** [reply]
These are the participation factor of the states (shown to the left) in the modes. The modes can be in any order, so they are referred to by numbers.

**Reply by Leonawyciii** [reply]
Once again, thank you for the information and for being so helpful.

**Comment by jinningwang** [comment]
Is this an answer for your question? https://docs.andes.app/en/latest/_generated/_generated/andes.routines.eig.EIG.export_mat.html?highlight=EIG
